src/ert/gui/simulation/view/update.py

```diff
@@ -127,7 +127,7 @@ def iteration(self) -> int:
     def _insert_status_message(self, message: str) -> None:
         item = QListWidgetItem()
         item.setText(message)
-        item.setFlags(item.flags() & ~Qt.ItemFlags(Qt.ItemFlag.ItemIsEnabled))
+        item.setFlags(item.flags() & ~Qt.ItemFlag.ItemIsEnabled)
```

Comment:
Can we use `item.setFlag` instead?

Reply:
there is no setFlag method on QListWidgetItem